cmd/forwarder/main.go

```diff
@@ -44,7 +44,7 @@ func main() {
 	forwarderSet := newForwarderSet(config)
 
 	shutdownCh := make(shutdownCh)
-	httpServer := newHTTPServer(config, auth, fix, forwarderSet)
+	httpServer := newHTTPServer(config, auth, logplexToSyslog, forwarderSet)
```

Comment:
I know I'm missing something here, but I'm having trouble understanding why only `logplexToSyslog` is referenced when setting up the http server. I'm assuming that `go forwarderSet.Run()` below is going to setup the primary handler you're modifying above, I'm just a having a little trouble parsing it. I guess there's not really a question here, more mentioning incase there is something off here and helps you notice it.

Reply:
Thanks for calling this out. I didn't realize we were passing `logplexToSyslog` into `newHTTPServer()`. My guess is that it's there to ease testing, but I'll investigate and see what's up.